README.md

```diff
@@ -2306,6 +2306,34 @@ describe Twitter::API do
   end
 end
 ```
+#### Airborne
+
+You can test with [Airborne](https://github.com/brooklynDev/airborne) which uses `rack-test` to make requests. It then parses the response and provides helpers for testing JSON, including path matchers for testing nested JSON and arrays.
+
+```ruby
+#spec_helper.rb
+require 'airborne'
+Airborne.configure do |config|
+  config.rack_app = MyGrapeApp
+end
+```
+
+```ruby
+require 'airborne'
+
+describe 'sample spec' do
+  it 'validates types' do
+    get '/api/v1/simple_get' #json api that returns { "name" : "John Doe" }
```

Comment:
Make this as if Rubocop ran on it, so space after pound, `# json ...`, then there're spaces between ` { }` and the hash syntax below should be Ruby 1.9.3, so `name: 'John Doe'`.